### PR TITLE
CLDC-3127 Amend renewal question wording for 2024

### PR DESCRIPTION
**Pull request by kosiakkatrina**
Amend renewal question header and hint text for 2024 and disable collection year validations on review apps only so that these changes could be tested

### CODE REVIEW

File: app/models/form/lettings/questions/renewal.rb [hunk 1]
```diff
@@ -3,15 +3,32 @@ def initialize(id, hsh, page)
...
 
   ANSWER_OPTIONS = {
     "1" => { "value" => "Yes" },
     "0" => { "value" => "No" },
-  }.freeze
+  }
+  .freeze
+
+  def header_text
+    if form.start_date && form.start_date.year >= 2024
```
Comment (natdeanlewissoftwire) [inline app/models/form/lettings/questions/renewal.rb]: think it's worth saving this as a method as we're going to use this exact thing loads?

Comment (natdeanlewissoftwire) [reply app/models/form/lettings/questions/renewal.rb]: or can we use `&.`?

File: app/services/feature_toggle.rb [hunk 2]
```diff
@@ -1,7 +1,11 @@
 class FeatureToggle
   # Disable check on preview apps to allow for testing of future forms
   def self.saledate_collection_window_validation_enabled?
-    true
+    Rails.env.production? || Rails.env.test? || Rails.env.staging?
```
Comment (natdeanlewissoftwire) [inline app/services/feature_toggle.rb:4]: have you checked this won't have any unexpected changes on local envs?

**Review by natdeanlewissoftwire — APPROVED** [review]
lgtm! just thrown a few ideas out but have approved

File: app/services/feature_toggle.rb (hunk 2 above)
Comment (kosiakkatrina) [reply app/services/feature_toggle.rb:4]: Good point! It deffo removes the validation locally as well. It doesn't matter too much cause we can always toggle things on/off locally ourselves anyway, so any preference on this? I think it aligns with some other toggles we have atm

File: app/models/form/lettings/questions/renewal.rb (hunk 1 above)
Comment (kosiakkatrina) [reply app/models/form/lettings/questions/renewal.rb]: `&.` will still execute `>=` and it doesn't know what to do in `nil` situations. I've extracted it to form tho, let me know if that's better

**Review by natdeanlewissoftwire — APPROVED** [review]

File: app/services/feature_toggle.rb (hunk 2 above)
Comment (natdeanlewissoftwire) [reply app/services/feature_toggle.rb:4]: yeah fine as is by me :D

**Pull request merged**